### ISSUE TITLE
could henkaku firmware be installed?

### ISSUE THREAD

**Issue by justalex95** [Vita3K/Vita3K#2931]
Could henkaku firmware be installed on vita3k? mainly asking becausethings like vitagrafix and threadoptimser plugins could be really useful especially in terms of performance. I made threadoptimiser profiles a while ago for killzone and wipeout that improved performance to the tune of 10-20 extra fps using threadoptimiser to optimise the games processing threads in regards to cpu affinity and thread priority and it worked really well.

wondering if this could be a shortcut to improve performance if henkaku could b installed with plugins

**Comment by bookmist** [comment]
No, and it have no sense. Vita3k use OS thread sheduler and ignore cpu affinity and thread priority for thread sheduling.

**Comment by justalex95** [comment]
Ok, so what about removing the 30fps locks on roughly 50 games using vitagrafix? I would want that, right now they cant go over 30... wipeout 2048  killzone (I know it doesnt work right now but it has a 30 fps lock) the amazing spiderman....

surely thats a big plus for an emulator with hardware that could easily run these games at 60 at some point? if the plugins could work it would save alot of time for the developer if that became a feature of vita3k in the future because its based on a game by game basis and the work is already done for quite alot of games.

for 60 fps gameplay it seems like an idea. if the developer has other plans for this feature, then obviously its their baby, they know what they're doing and the progress they've made performance wise in the last 6 months to a year is pretty insane...

only putting it out there as an idea since 60 fps is where everyone wants to get to and I saw this as a possible solution

**Comment by Croden1999** [comment]
homebrew plugins are not supported on Vita3K, so, it is invalid

**Comment by justalex95** [comment]
ah well, thats a shame, what made the vita good was the plugins and the ability to change it from stock... made it a real console. appreciate the replies
